packages/devtools/lib/src/timeline/event_details.dart

```diff
@@ -208,26 +206,26 @@ class _UiEventDetails extends CoreElement {
     switch (tabType) {
       case EventDetailsTabType.flameChart:
         flameChart.show();
-        bottomUp.hide();
-        callTree.hide();
+        bottomUp.hidden(true);
+        callTree.hidden(true);
         break;
       case EventDetailsTabType.bottomUp:
-        flameChart.hide();
+        flameChart.hidden(true);
         bottomUp.show();
-        callTree.hide();
+        callTree.hidden(true);
         break;
       case EventDetailsTabType.callTree:
-        flameChart.hide();
-        bottomUp.hide();
+        flameChart.hidden(true);
+        bottomUp.hidden(true);
         callTree.show();
         break;
     }
   }
 
   void hideAll() {
-    flameChart.hide();
-    bottomUp.hide();
-    callTree.hide();
+    flameChart.hidden(true);
```

Comment:
this seems like just the default case if no tab is selected.

Reply:
A tab will always be selected. Right now, the flame chart tab is set as the default selected tab in the view.